### PR TITLE
[15.0][IMP] bank-payment : test performance improvement

### PR DESCRIPTION
**Pull request by josep-tecnativa**
CC @Tecnativa TT45458

### CODE REVIEW

**Comment by josep-tecnativa** [comment]
ping @pedrobaeza

File: account_banking_mandate/tests/test_invoice_mandate.py [hunk 1]
```diff
@@ -197,6 +197,20 @@ def _create_res_bank(self, name, bic, city, country):
...
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
+
     def setUp(self):
```
Comment (pedrobaeza) [inline account_banking_mandate/tests/test_invoice_mandate.py]: You have to replace this one.

File: account_payment_order/tests/test_payment_mode.py [hunk 2]
```diff
@@ -9,62 +9,72 @@
...
+                tracking_disable=True,
+            )
+        )
         Method_get_payment_method_information = (
             AccountPaymentMethod._get_payment_method_information
         )
 
-        def _get_payment_method_information(self):
-            res = Method_get_payment_method_information(self)
+        def _get_payment_method_information(cls):
```
Comment (pedrobaeza) [inline account_payment_order/tests/test_payment_mode.py:29]: When this is something called from `setUpClass`, it should be a class method

```suggestion
        @classmethod
        def _get_payment_method_information(cls):
```

File: account_banking_mandate_contact/tests/test_account_payment_order.py [hunk 3]
```diff
@@ -10,6 +10,20 @@
...
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
+
     def setUp(self):
```
Comment (pedrobaeza) [inline account_banking_mandate_contact/tests/test_account_payment_order.py]: And this one?

**Comment by pedrobaeza** [comment]
Remove also empty lines inside the method.

File: account_banking_mandate_contact/tests/test_account_payment_order.py (hunk 3 above)
Comment (josep-tecnativa) [reply account_banking_mandate_contact/tests/test_account_payment_order.py]: I' am on it.

**Comment by josep-tecnativa** [comment]
ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
OK, good enough

/ocabot merge patch

**Pull request merged**